app/javascript/packages/document-capture/components/in-person-full-address-search.tsx

```diff
@@ -63,11 +63,14 @@ function useUspsLocations(locationsURL: string) {
       validatedZipCodeFieldRef.current?.setCustomValidity('');
       validatedZipCodeFieldRef.current?.reportValidity();
 
+      const validZipCodeLength = zipCodeValue.length === 5 || zipCodeValue.length === 10;
+
       if (
-        addressValue.trim().length === 0 ||
```

Comment:
I removed .trim() for each of these values. The values for each of these fields have been passed into this function already trimmed. I also added an additional check so that we don't call off to USPS when zipCodeValue is not 0 but is not valid. I noticed we were making this call with an invalid zip code length.